### PR TITLE
cgroup v2 cleanups

### PR DESCRIPTION
**Pull request by kolyshkin**
This is a set of small cleanups and fixes for cgroup v2 unified hierarchy + systemd. Please see individual commits for descriptions.

@AkihiroSuda PTAL

### CODE REVIEW

**Comment by kolyshkin** [comment]
I hope this is very reviewable and can be merged soon. I have more stuff in the same direction

File: libcontainer/cgroups/systemd/unified_hierarchy.go [hunk 1]
```diff
@@ -21,6 +20,8 @@ import (
 	"github.com/sirupsen/logrus"
 )
 
+const UnifiedMountpoint = fs2.UnifiedMountpoint
```
Comment (AkihiroSuda) [inline libcontainer/cgroups/systemd/unified_hierarchy.go]: Can we just directly use `fs2.UnifiedMountpoint` without aliasing?

File: libcontainer/cgroups/systemd/unified_hierarchy.go [hunk 2]
```diff
@@ -197,14 +189,26 @@ func (m *UnifiedManager) GetUnifiedPath() (string, error) {
...
+}
+
 func createCgroupsv2Path(path string) (Err error) {
 	content, err := ioutil.ReadFile("/sys/fs/cgroup/cgroup.controllers")
 	if err != nil {
 		return err
 	}
-	if !filepath.HasPrefix(path, "/sys/fs/cgroup") {
-		return fmt.Errorf("invalid cgroup path %s", path)
-	}
```
Comment (AkihiroSuda) [inline libcontainer/cgroups/systemd/unified_hierarchy.go]: Why remove?

**Comment by AkihiroSuda** [comment]
A couple of nits, but overall LGTM

File: libcontainer/cgroups/systemd/unified_hierarchy.go (hunk 1 above)
Comment (kolyshkin) [reply libcontainer/cgroups/systemd/unified_hierarchy.go]: Sure, I just thought it's cleaner that way. Will fix

File: libcontainer/cgroups/systemd/unified_hierarchy.go (hunk 2 above)
Comment (kolyshkin) [reply libcontainer/cgroups/systemd/unified_hierarchy.go]: We're calling this function from a single place in the same file with path from `getv2Path()` which clearly sets prefix to `/sys/fs/cgroup/`.

I can reinstate the check but it seems redundant now.

File: libcontainer/cgroups/systemd/unified_hierarchy.go (hunk 1 above)
Comment (kolyshkin) [reply libcontainer/cgroups/systemd/unified_hierarchy.go]: fixed

**Comment by AkihiroSuda** [comment]
LGTM

**Comment by mrunalp** [comment]
LGTM

**Pull request merged**